src/model/openthaigpt_pretraining_model/models/nanoGPT/model.py

```diff
@@ -28,8 +68,1017 @@ def _attn_wrapper(self, query, key, value, attention_mask=None, head_mask=None):
     return attn_out, None
 
 
+def load_tf_weights_in_gpt2(model, config, gpt2_checkpoint_path):
```

Comment:
You are copied this whole function from [huggingface gpt2](https://github.com/huggingface/transformers/blob/main/src/transformers/models/gpt2/modeling_gpt2.py). Please import the function from huggingface.